src/module_loader.rs

```diff
@@ -126,10 +158,21 @@ impl ModuleLoader for RustyLoader {
         &self,
         specifier: &str,
         referrer: &str,
-        _kind: deno_core::ResolutionKind,
+        kind: deno_core::ResolutionKind,
     ) -> Result<ModuleSpecifier, anyhow::Error> {
         // Resolve the module specifier to an absolute URL
         let url = deno_core::resolve_import(specifier, referrer)?;

```

Comment:
Is it possible to create a custom url resolver for a module specifier?

I want to make imports look like `... from 'my-module'` to be compatible with Node.js. However, I'm encountering an error from [here](https://github.com/denoland/deno_core/blob/f217fe0a6730bfef2a7f2a75fa32659d71d80a95/core/module_specifier.rs#L69):
```
Relative import path "my-module" not prefixed with / or ./ or ../ from "file:///<...>/index.js"
```